test/helpers/codec/encoding.ts

```diff
@@ -26,12 +26,18 @@ export interface SignatureParameters {
   s: string
 }
 
+/**
+ * @dev Trace where `DEFAULT_EIP155_TX` is called.
+ */
 export const DEFAULT_EIP155_TX: EIP155Transaction = {
   to: `0x${'12'.repeat(20)}`,
   nonce: 100,
   gasLimit: 1000000,
   gasPrice: 100000000,
   data: `0x${'99'.repeat(10)}`,
+  /**
+   * @dev Encoding `chainId`
+   */
```

Comment:
this comment feels a bit disruptive, could we just append it after the line? ie.

`chainId: 420, // Encoding chainId`